packager/packager.cc

```diff
@@ -502,6 +532,14 @@ struct PackageInfoFinder {
             info = make_unique<PackageInfoImpl>();
             checkPackageName(ctx, nameTree);
             info->name = getPackageName(ctx, nameTree);
+
+            // Append _Private at the end of the mangled package name to get the name of its private namespace.
+            auto utf8PrivateName =
+                ctx.state.enterNameUTF8(absl::StrCat(info->name.mangledName.show(ctx.state), "_Private"));
+            auto packagerPrivateName =
+                ctx.state.freshNameUnique(core::UniqueNameKind::PackagerPrivate, utf8PrivateName, 1);
+            info->privateMangledName = ctx.state.enterNameConstant(packagerPrivateName);
+
```

Comment:
Since `getPackageName` is also -- in fact mostly -- called in the context of querying imports (in which the private mangled name is not necessary), I wanted to save an additional allocation.